src/Samples/SarifToCsv/Program.cs

```diff
@@ -150,6 +149,13 @@ public static void ConvertSarifLog(JsonSerializer serializer, string sarifFilePa
             writers["Location.Region.StartLine"] = (writer, result, pLoc) => { writer.Write(pLoc.Region?.StartLine ?? -1); };
             writers["Location.Region.Tags"] = (writer, result, pLoc) => { writer.Write(String.Join("; ", ((IEnumerable<string>)pLoc.Region?.Tags) ?? Array.Empty<string>())); };
 
+            writers["Properties.MatchState"] = (writer, result, pLoc) => 
```

Comment:
>Properties.MatchState [](start = 21, length = 21)

it doesn't make sense to me that we would emit some private match state from properties and yet not provide suppressions information from the core SARIF types.

Reply:
I'll take this out. This was to support my EngineTest comparisons quickly. I need to add support for collection properties in general in order to really handle the remaining result properties.